src/operator/OrgOverlay.tsx

```diff
@@ -209,7 +216,7 @@ const OrgOverlay: FC = () => {
                   </Grid.Column>
                   <Grid.Column widthMD={Columns.Three}>
                     <Form.Label label="Blocked Notification Endpoints" />
-                    <LimitsInput
+                    <LimitsField
```

Comment:
related to the comment above, we're never using the `type` so i'd be in favor of reducing abstraction and just inlining this stuff and removing the use of `lodash` in that component